app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
In case your closing parenthesis does not have any arguments on the same line, your opening parenthesis should not have any arguments on the same line as well. Place each argument on a new line. Make this change in the `CarWashStation` class as well.